changelogs/current.yaml

```diff
@@ -192,4 +192,9 @@ new_features:
     Added :ref:`delay_deny <envoy_v3_api_msg_extensions.filters.network.rbac.v3.RBAC>` to support deny connection after
     the configured duration.
 
+- area: cluster
```

Comment:
please remove blank line above this one.

Reply:
not sure why the blank line below before `deprecated:` was removed, and not the line above this one. Can you please take another look?

Reply:
my mistake, fixed